stan/math/prim/fun/read_corr_L.hpp

```diff
@@ -36,6 +36,12 @@ template <typename T>
 Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> read_corr_L(
     const Eigen::Array<T, Eigen::Dynamic, 1>& CPCs,  // on (-1, 1)
     size_t K) {
```

Comment:
[question]
Given that there's a `K == 0` case in other situations, why not here?

Reply:
I didn't put it in `read_corr_L` originally because the `K == 0` was already handled in `read_corr_matrix`. I'm now adding it to `read_corr_L` too, as this function is called also from other places that don't seem to do that check.